flow-server/src/main/java/com/vaadin/flow/server/Constants.java

```diff
@@ -412,6 +412,8 @@
     // Non-default port currently not supported (#7970)
     public static final int SPRING_BOOT_DEFAULT_LIVE_RELOAD_PORT = 35729;
 
+    public static final String VAADIN_ENDPINT_ENABLED = "endpoint.enabled";
```

Comment:
```suggestion
    public static final String VAADIN_ENDPOINT_ENABLED = "endpoint.enabled";
```
There's a typo in the constant. BTW, should it be a public constant (from API maintenance point of view this means it can't easily be removed or renamed later) or could it be put inside `com.vaadin.flow.server.connect` (e.g., within an `internal` package)?

Reply:
Very good point, instead of create a new internal package inside `connect`, I reused a flow internal package and added a new InternalConstants, please help to take another look

Reply:
👍. Looks like the imports may need updating.

Reply:
Good spot, thanks. Thought VS Code would have done that automatically for me.